okhttp/src/main/java/okhttp3/internal/ws/WebSocketOptions.kt

```diff
@@ -0,0 +1,57 @@
+package okhttp3.internal.ws
+
+import okhttp3.Response
+import java.io.IOException
+
+data class WebSocketOptions(
```

Comment:
Please make sure you’ll cover all this with tests.

Reply:
Well, sure, just gathering feedback on the approach now.